packages/server/src/fhir/operations/evaluatemeasure.ts

```diff
@@ -1,18 +1,41 @@
-import { Operator, badRequest, created, parseSearchRequest } from '@medplum/core';
+import { Operator, created, parseSearchRequest } from '@medplum/core';
+import { FhirRequest, FhirResponse } from '@medplum/fhir-router';
 import {
   Measure,
   MeasureGroup,
   MeasureGroupPopulation,
   MeasureReport,
   MeasureReportGroup,
   MeasureReportGroupPopulation,
-  Resource,
+  OperationDefinition,
 } from '@medplum/fhirtypes';
-import { Request, Response } from 'express';
 import { getAuthenticatedContext } from '../../context';
-import { sendOutcome } from '../outcomes';
 import { Repository } from '../repo';
-import { isFhirJsonContentType, sendResponse } from '../response';
+import { parseInputParameters } from './utils/parameters';
+
+const operation: OperationDefinition = {
+  resourceType: 'OperationDefinition',
+  id: 'Measure-evaluate-measure',
+  url: 'http://hl7.org/fhir/OperationDefinition/Measure-evaluate-measure',
+  name: 'Evaluate Measure',
+  status: 'draft',
+  kind: 'operation',
+  code: 'evaluate-measure',
+  resource: ['Measure'],
+  system: false,
+  type: true,
+  instance: true,
+  parameter: [
+    { name: 'periodStart', use: 'in', min: 1, max: '1', type: 'date' },
+    { name: 'periodEnd', use: 'in', min: 1, max: '1', type: 'date' },
+    { name: 'measure', use: 'in', min: 0, max: '1', type: 'Reference' },
+    { name: 'reportType', use: 'in', min: 0, max: '1', type: 'code' },
+    { name: 'subject', use: 'in', min: 0, max: '1', type: 'Reference' },
+    { name: 'practitioner', use: 'in', min: 0, max: '1', type: 'Reference' },
+    { name: 'lastReceivedOn', use: 'in', min: 0, max: '1', type: 'dateTime' },
+    { name: 'return', use: 'out', min: 1, max: '1', type: 'MeasureReport' },
+  ],
+};
```

Comment:
Why not use the [actual FHIR spec definition](http://hl7.org/fhir/R4/measure-operation-evaluate-measure.html) of this one?

```ts
const operation = getOperationDefinition('Measure', 'evaluate-measure');
```

Reply:
Oh, nice.